src/foundry/client-esm/applications/api/application.d.mts

```diff
@@ -403,14 +416,35 @@ declare class ApplicationV2<
   [__RenderOptions]: RenderOptions;
   [__RenderContext]: RenderContext;
 
+  /**
+   * Applications are constructed by providing an object of configuration options.
+   * @param options - Options used to configure the Application instance
+   */
   constructor(options?: DeepPartial<Configuration>);
```

Comment:
Please check null (as a reminder you can just add `// not: null` if it's a parameter default etc.)